fastify.js

```diff
@@ -206,8 +206,10 @@ function build (options) {
     if (isAbstract === false) {
       res._startTime = now()
     }
-    res.on('finish', onResFinished)
-    res.on('error', onResFinished)
+    if (isAbstract === false || context.onResponse !== null) {
```

Comment:
You can do the same for `res._context = context` :)

Reply:
done